paws.common/R/iniutil.R

```diff
@@ -0,0 +1,33 @@
+# Get the profile name from an ini file
+extract_ini_profile <- function(item) {
+  profile <- gsub("\\[|\\]", "", item)
```

Comment:
I would make this `"^\\[.*\\]"` to make sure it only captures lines beginning with `[`.